src/tests/system/tests/test_ldap.py

```diff
@@ -240,3 +249,271 @@ def test_ldap__password_change(client: Client, ldap: LDAP, method: str):
 
     # Authenticate with new password
     assert client.auth.parametrize(method).password("tuser", "Redhat@321")
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(bz=1928648)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_ldap__network_timeout_parameters_shown_in_logs(client: Client, ldap: LDAP):
```

Comment:
During review it would be slightly easier if a comment was included noting where a test originally came from if it was moved from another file.

Reply:
I don't want that merged in. It's in the google sheets though.